pkg/hypervctl/vmm.go

```diff
@@ -4,17 +4,23 @@
 package hypervctl
 
 import (
-	"errors"
-	"fmt"
+	"strconv"
+	"strings"
 
-	"github.com/containers/libhvee/pkg/wmiext"
+	"github.com/containers/libhvee/pkg/powershell"
 )
 
-const (
-	HyperVNamespace                = "root\\virtualization\\v2"
-	VirtualSystemManagementService = "Msvm_VirtualSystemManagementService"
-	MsvmComputerSystem             = "Msvm_ComputerSystem"
-)
+
+func CheckHypervAvailable() error {
+	if !powershell.IsHypervAdministrator() {
+		return powershell.ErrNotAdministrator
+	}
+	if err := powershell.HypervAvailable(); err != nil {
+		return err
+	}
```

Comment:
It should work the same but shouldn't the check be in opposite order? First, check if hyperv is available and then if the user is in group? Just a logical preference